src/registry/utils/spaces-to-underscores.ts

```diff
@@ -0,0 +1,3 @@
+export default function convertSpacesToUnderscores(input: string) {
```

Comment:
same here https://github.com/webdoky/yari-ports/blob/master/lib/kuma/utils/spaces-to-underscores.js